src/integTest/groovy/com/bmuschko/gradle/docker/DockerJavaApplicationPluginIntegrationTest.groovy

```diff
@@ -137,6 +137,23 @@ class DockerJavaApplicationPluginIntegrationTest extends AbstractIntegrationTest
         tags.first() == testTagName
     }
 
+    def "Can configure the dockerJava.javaApplication extension dynamically and set multiple tags"() {
+        given:
+        String testTagName1 = "some-test-tag"
+        String testTagName2 = "another-test-tag"
+        applyDockerJavaApplicationPluginAndApplicationPlugin(project)
+        when:
+        project.docker.javaApplication {
+            tags.set([testTagName1, testTagName2])
+        }
+        then:
+        DockerBuildImage task = project.tasks.findByName(DockerJavaApplicationPlugin.BUILD_IMAGE_TASK_NAME)
+        Set<String> tags = task.tags.get()
+        tags.size() == 2
+        tags.contains(testTagName1)
```

Comment:
You can probably just say `tags == [testTagName1, testTagName2]` here to make it simpler.